packages/components/fields/radio-field/src/radio-field.spec.js

```diff
@@ -178,21 +178,27 @@ describe('when showing an info button', () => {
 describe('when field is touched and has errors', () => {
   describe('when field empty', () => {
     it('should render a default error', () => {
-      const { getByText } = renderRadioField({
+      const { container, getByText } = renderRadioField({
         touched: true,
         errors: { missing: true },
       });
       expect(getByText(/field is required/i)).toBeInTheDocument();
+      expect(
+        container.querySelector('[aria-invalid="true"]')
+      ).toBeInTheDocument();
```

Comment:
`Radio.Group` component adds the aria error attributes to a wrapper `div` of the radio inputs. That's why I used a query selector.